db/migrate/20140828084844_add_vmt_to_migration_status.rb

```diff
@@ -0,0 +1,23 @@
+class AddVmtToMigrationStatus < ActiveRecord::Migration
+  def change
+    add_column :atmosphere_migration_jobs,
+               :virtual_machine_template_id,
+               :int
+
+    add_foreign_key :atmosphere_migration_jobs,
+                    :atmosphere_virtual_machine_templates,
+                    column: 'virtual_machine_template_id',
+                    name: 'atmo_mj_vmt_fk'
+
+    remove_index :atmosphere_migration_jobs,
```

Comment:
I was removing default creating index and create a new one.

I don't know why. Maybe I copied this solution from other migration or it was my creation.